dev/checkstyle.xml

```diff
@@ -168,5 +182,6 @@
         <module name="UnusedImports"/>
         <module name="RedundantImport"/>
         <module name="RedundantModifier"/>
+        <module name="FileContentsHolder"/>
```

Comment:
@srowen `SuppressionCommentFilter`and `FileContentsHolder` must work together,http://stackoverflow.com/questions/5761188/checkstyle-suppressioncommentfilter-not-ignoring-specified-rule